.github/workflows/main.yml

```diff
@@ -78,6 +79,10 @@ jobs:
           # Returns 'true' if the input platform list matches any of the platform monikers given as argument,
           # 'false' otherwise.
           # arg $1: platform name or names to look for
+
+          # List of platforms to exclude by default
+          EXCLUDED_PLATFORMS=("alpine-linux-x64")
```

Comment:
I'm not to fond of declaring a list like this in the middle of a code body. Perhaps you can set it up as a non-required workflow input with alpine as the default value, declared next to the default list of platforms?

Reply:
yeah this makes sense, I'll make that change now

Reply:
@magicus do you want me to push this change?